lid.cpp

```diff
@@ -48,6 +37,89 @@ void Lid::validate(std::string filePath)
     return;
 }
 
+void Lid::assembleCodeUpdateImage()
+{
+    info("InbandCodeUpdate: assembleCodeUpdateImage\n");
+    utils::subscribeToSystemdSignals(bus);
+    auto method = bus.new_method_call(SYSTEMD_BUSNAME, SYSTEMD_PATH,
+                                      SYSTEMD_INTERFACE, "StartUnit");
+    method.append("assemble-lids.service", "fail");
```

Comment:
I couldn't find a "fail" argument to the "StartUnit" method. Should it be "replace" instead? Seems that's how service units are started.

Reply:
done